### PR TITLE
SOV-1758: Implement new notification signup user flow

### PR DESCRIPTION
**Pull request by tiltom**
Resolves https://sovryn.atlassian.net/browse/SOV-1758

### CODE REVIEW

**Review by pietro-maximoff — APPROVED** [review]

File: apps/frontend/src/locales/en/translations.json [hunk 1]
```diff
@@ -572,7 +572,11 @@
         "successMessage": "Your preferences have been saved.",
         "errorMessage": "Something went wrong, please try again.",
         "authErrorMessage": "Authentication failed, please try again.",
```
Comment (soulBit) [inline apps/frontend/src/locales/en/translations.json]:
```suggestion
        "authErrorMessage": "Authentication failed",
```

Needs to be updated as per spec - I'm fine with current copy if it's approved by John (but we'll have to update D1 PR to match if so)

File: apps/frontend/src/locales/en/translations.json [hunk 2]
```diff
@@ -572,7 +572,11 @@
...
-        "unconfirmedEmailWarning": "Unconfirmed - check inbox for confirmation email"
+        "unconfirmedEmailWarning": "Unconfirmed - check inbox for confirmation email",
+        "invalidEmail": "Invalid email address",
+        "unsavedChanges": "Your changes have not been saved yet",
+        "savedChanges": "Changes saved",
+        "unsubscribed": "You were successfully unsubscribed"
```
Comment (soulBit) [inline apps/frontend/src/locales/en/translations.json]:
```suggestion
        "unsubscribed": "Unsubscribed"
```

Needs to be updated as per spec - I'm fine with current copy if it's approved by John (but we'll have to update D1 PR to match if so)

**Review by soulBit — CHANGES_REQUESTED** [review]
Two minor things to check with Product

File: apps/frontend/src/locales/en/translations.json (hunk 2 above)
Comment (tiltom) [reply apps/frontend/src/locales/en/translations.json]: I committed the changes, no need to overcomplicate it and we can always change it in the future.

**Review by soulBit — APPROVED** [review]
Looks good tried on test/mainnet and works well, only part that I didn't explicitly test was point 4 from spec - going to confirm with Shadow if that's been implemented as per spec. edit: confirmed that was tested thoroughly on testnet already, can't test on mainnet unless system is in critical state

**Pull request merged**